sdk-core/src/main/java/care/data4life/sdk/network/interceptor/InterceptorHelper.kt

```diff
@@ -13,8 +13,13 @@
  * applications and/or if you’d like to contribute to the development of the SDK, please
  * contact D4L by email to help@data4life.care.
  */
-package care.data4life.sdk
 
-interface NetworkConnectivityService {
-    val isConnected: Boolean
+package care.data4life.sdk.network.interceptor
+
+import okhttp3.Request
+
+fun Request.Builder.replaceHeader(name: String, value: String): Request.Builder {
+    return this
+        .removeHeader(name)
+        .addHeader(name, value)
```

Comment:
Nope...but you are right pointing on this in other way...the builder has a method called `header`, which actually does the replacement - I will change that, but I like to keep the `replaceHeader` then as a alias, since `header` as a method name sounds not expressive - are you okay with that?

Reply:
i am. thanks

Reply:
done